antarest/login/web.py

```diff
@@ -287,84 +299,29 @@ def roles_delete(user: int, group: str) -> Any:
     @bp.route("/bots", methods=["POST"])
     @auth.protected()
     def bots_create() -> Any:
-        """
-        Create Bot
-        ---
-        responses:
-          '200':
-            content:
-              application/json:
-                schema:
-                  type: string
-                  description: Bot token API
-            description: Successful operation
-          '400':
-            description: Invalid request
-          '401':
-            description: Unauthenticated User
-          '403':
-            description: Unauthorized
-        consumes:
-            - application/json
-        parameters:
-        - in: body
-          name: body
-          required: true
-          description: Bot
-          schema:
-            id: User
-            required:
-                - name
-                - group
-                - role
-            properties:
-                name:
-                    type: string
-                    description: Bot name
-                isAuthor:
-                    type: boolean
-                    description: Set Bot impersonator between itself or it owner
-                group:
-                    type: string
-                    description: group id linked to bot
-                role:
-                    type: int
-                    description: RoleType used by bot. Should be lower or equals ot owner role type inside same group
-        tags:
-          - Bot
-        """
         params = RequestParameters(user=Auth.get_current_user())
-        create = BotCreateDTO.from_dict(json.loads(request.data))
+        create = BotDTO.from_dict(json.loads(request.data))
         bot = service.save_bot(create, params)
 
         if not bot:
             return UserHasNotPermissionError()
 
-        group = service.get_group(create.group, params)
-        if not group:
-            return UserHasNotPermissionError()
-
-        jwt = JWTUser(
-            id=bot.id,
-            impersonator=bot.get_impersonator(),
-            type=bot.type,
-            groups=[JWTGroup(id=group.id, name=group.name, role=create.role)],
-        )
-        tokens = generate_tokens(jwt, expire=timedelta(days=368 * 200))
-        return tokens["access_token"]
```

Comment:
The idea of this was to generate a bot once and return (without saving it) the token that will be displayed once to the user (he has to write it somewhere).